src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/DefaultHttpClient.java

```diff
@@ -17,63 +17,77 @@
 import okhttp3.*;
 
 final class DefaultHttpClient extends HttpClient {
-    private final OkHttpClient client;
+    private OkHttpClient client = null;
 
     public DefaultHttpClient() {
-        this(5000);
+        this(5*1000, null);
```

Comment:
Weird. If you like 5*1000 over 5000, maybe you'd like 5_000 better yet.

Reply:
I didn't know that was a thing lol. This can probably just be 5000. I just like that pattern for milliseconds